src/gfx.cpp

```diff
@@ -1185,6 +1190,50 @@ static void GfxBlitter(const Sprite * const sprite, int x, int y, BlitterMode mo
 	BlitterFactory::GetCurrentBlitter()->Draw(&bp, mode, zoom);
 }
 
+/**
+ * Draws a sprite to a new RGBA buffer (see Colour union) instead of drawing to the screen.
+ * Returns a nullpointer if a 8bpp blitter is being used.
```

Comment:
No other occurrence of `nullpointer` exists in the rest of the code base.
Remove this line, and instead replace `@return Pixel Buffer` with `@return Pixel buffer, or nullptr if an 8bpp blitter is being used.`